src/components/Filters/StaticFiltersProvider.tsx

```diff
@@ -28,7 +28,7 @@ export type StaticFiltersProviderProps = PropsWithChildren<{
  */
 export function StaticFiltersProvider({
   children,
-  className = 'md-56',
```

Comment:
for my edification what was md-56 doing before? did we just miss a breaking change when we upgraded to tailwind 3 or something?

Reply:
tried looking for it but couldn't find anything :(

Reply:
I believe this was a typo. `md-56` does not apply any additional css to the div. md sets breakpoint (768px) of when the styling on the right hand side of the colon should be applied.
![Screen Shot 2022-07-06 at 11 45 35 AM](https://user-images.githubusercontent.com/36055303/177591236-9b7724d8-1345-4bb1-8027-94c3e811f62a.png)

Reply:
awesome!